spec/index.bs

```diff
@@ -7142,22 +7142,21 @@ dictionary GPURenderPassDepthStencilAttachment {
     Given a {{GPURenderPassDepthStencilAttachment}} |this| the following validation
     rules apply:
 
-    - |this|.{{GPURenderPassDepthStencilAttachment/view}} must have a [=depth or stencil renderable
+    1. |this|.{{GPURenderPassDepthStencilAttachment/view}} must have a [=depth or stencil renderable
```

Comment:
why are we changing the "-" into "1" here? The list is not ordered.

Reply:
I noticed that the validation rules for RenderPassDesc and ColorAttachment are in ordered list. But it is not necessary to be ordered, and the majority of validation rules in the spec are not in ordered list (Although they are not always consistent). So I recovered this to what it is. My bad...